app/src/main/res/values/strings.xml

```diff
@@ -480,4 +480,14 @@
   <string name="developer_options_force_crash">Force crash</string>
   <string name="all">All</string>
   <string name="mark_completed_uppercase">MARK COMPLETED</string>
+  <!-- ThirdPartyDependencyListActivity -->
+  <string name="third_party_dependency_list_activity_title">Third-party Dependencies</string>
+  <string name="third_party_dependencies">Third Party Dependencies</string>
+  <string name="third_party_dependency_version">v %s</string>
```

Comment:
```suggestion
  <string name="third_party_dependency_version">version %s</string>
```

Is this what you're going for? 'v' probably won't translate well, or be read correctly by screenreaders.

Reply:
Change done, now the UI looks like this - 
<img src="https://user-images.githubusercontent.com/54636525/128175128-d3e74b5f-b92a-4ee3-96eb-83416c2e54f6.jpg" width=300 />)

Reply:
I think that looks reasonable. @rt4914 any thoughts?

Reply:
From translation perspective it completely makes sense to use `version` instead of `v`, Thanks a lot.

Reply:
Sounds good @rt4914.